sdk/apimanagement/apimanagement-custom-widget-scaffolder/review/apimanagement-custom-widget-scaffolder.api.md

```diff
@@ -0,0 +1,52 @@
+## API Report File for "@azure-tools/apimanagement-custom-widget-scaffolder"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public (undocumented)
+export function generateProject(widgetConfig: TWidgetConfig, deployConfig: TDeployConfig, { openUrl }?: TMiscConfig): Promise<void>;
+
+// @public (undocumented)
+export interface TCustomWidgetConfig extends TWidgetConfig {
+    // (undocumented)
+    deployed?: string;
+    // (undocumented)
+    override?: string | boolean;
```

Comment:
What string values do these take? Their name sound like booleans so I wonder if we can just define them as a boolean?

Reply:
deployed is timestamp when it was deployed and override optionally contains from where is the content overridden from